lib/faction/setFactionSize.ts

```diff
@@ -1,33 +1,20 @@
+import { getRolledFromTable, ThresholdTable } from '../src/rollFromTable'
+import { assert } from '../src/utils'
 import { logger } from '../logger'
 import { Faction } from './_common'
-
-interface Town {
```

Comment:
don't know what's with this Town interface

Reply:
Might be to satisfy the needs of this module. The entire town object may not exist at this point, do this just ensures that it contains what this module needs. It should probably be called something else of that's the case.